v2/models/event.go

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// Event represents a single measurable event read from a device
+type Event struct {
```

Comment:
Why are we adding new v2 versions of our internal models? This doesn't look like it's changed from the existing model?

Reply:
As in the community core working group discussion we'll define separating models for v2 API to avoid confusion.